e2e/text-editor.e2e-spec.ts

```diff
@@ -68,11 +69,9 @@ describe('Text editor', () => {
       )).click();
       await element(by.css('.sky-dropdown-item button')).click();
       await SkyHostBrowser.moveCursorOffScreen();
-      setTimeout(() => {
-        expect('#screenshot-text-editor').toMatchBaselineScreenshot(done, {
-          screenshotName: getScreenshotName('text-editor-merge-field')
-        });
-      }, 1000);
+      expect('#screenshot-text-editor iframe').toMatchBaselineScreenshot(done, {
+        screenshotName: getScreenshotName('text-editor-merge-field')
+      });
```

Comment:
This test is not part of the scope of this PR.

The screenshot was sometimes showing focus state on the button, and sometimes not - causing the build to inconsistently fail. I realized, this test has nothing to do with the actual toolbar and buttons, so I focused the screenshot on the iframe, which avoids the problem entirely.